### DISCUSSION TITLE
Adyen Pix

### DISCUSSION THREAD

**Discussion by luizaraujobliver** [juspay/hyperswitch#3804] [General]
Dear all, we were informed by the slack channel that the pix was already available in the opensource version for adyen, but it does not appear in the payment methods to be activated.

**Comment by swangi-kumari** [comment]
Hey @luizaraujobliver ,
PIX for Adyen is already there in 2024.02.16.0 version.
On Hyperswitch Dashboard you can see this under Bank Transfer.
![Screenshot 2024-02-22 at 12 24 04 PM](https://github.com/juspay/hyperswitch/assets/85639103/ec250b43-d2a9-46fa-a7cf-2127c6725962)

**Reply by luizaraujobliver** [reply]
@swangi-kumari 
How do I test Pix with the Web SDK? It does not accept the Brazilian currency where Pix is accepted.

**Reply by luizaraujobliver** [reply]
@swangi-kumari We are using version 2024.02.21 which is ahead of 2024.02.16.0 and Pix does not appear. Is it necessary to make any configuration in the code for Pix to appear in the control center?

**Reply by swangi-kumari** [reply]
Hey @luizaraujobliver,
For PIX we need to configure, `currency = "BRL"` and `country = "BR"`.  And please try it again.

**Comment by luizaraujobliver** [comment]
@swangi-kumari  I don't mean when viewing at checkout. The option to activate Pix in the control center simply does not appear, we are using version 2024.02.21 which is ahead of 2024.02.16.0

**Reply by swangi-kumari** [reply]
@luizaraujobliver Can you please send the screenshot, for reference.

**Reply by luizaraujobliver** [reply]
@swangi-kumari  The Pix option for activation does not appear on Adyen. 
![image](https://github.com/juspay/hyperswitch/assets/123188775/63806aef-629f-4be2-bcb3-5f6368643834)

**Reply by swangi-kumari** [reply]
@luizaraujobliver  Are you using self hosted version of dashboard?
If yes, Can you please make sure the WASM build is in parity with the latest version of your dashboard.
